package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@brightsign/bsc",
-  "version": "1.0.1",
+  "name": "bsc",
```

Comment:
I'd leave this to match current npm package name

Reply:
Good catch - that was not intentional.